person/views.py

```diff
@@ -86,9 +78,18 @@ def get_context_data(self, **kwargs):
                 command_chain = {}
 
                 last_cited = repr(membership.object_ref.lastciteddate.get_value().value)
-                node_list = get_org_hierarchy_by_id(org.uuid, when=last_cited)
+                node_list_raw = get_org_hierarchy_by_id(org.uuid, when=last_cited)
                 edge_list = get_command_edges(org.uuid, when=last_cited)
 
+                # Add a unique URL to individual nodes for redirect to organization detail view
+                node_list = []
+                for node in node_list_raw:
+                    detail_id = node['detail_id']
+                    url = self.request.build_absolute_uri('/') + self.request.LANGUAGE_CODE + '/organization/detail/' + detail_id + '/'
```

Comment:
I think a more robust approach here would be to use Django's built-in [`reverse_lazy`](https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse-lazy) function for resolving URLs:

```python
url = reverse_lazy('detail_organization', args=(detail_id))
```

Reply:
Yes yes!